lib/rate_limiting.rb

```diff
@@ -119,7 +119,7 @@ def apply_rule(request, rule)
     key = rule.get_key(request)
     if cache_has?(key)
       record = cache_get(key)
-      logger.info "Rate Limit ENTRY #{request.url}: '#{key}' => #{record}"
+      logger.info "Rate Limit ENTRY #{request.path}: '#{key}' => #{record}"
```

Comment:
These are nice to have in production (and staging), but they won't show any params at all now.